### ISSUE TITLE
Allow customising retry delay and amount

### ISSUE THREAD

**Issue by markwylde** [rqlite/rqlite#653]
When starting rqlite and trying to join to another node, it will try 3 times over 15 seconds.

This seems to be configured [here](https://github.com/rqlite/rqlite/blob/5c8fb9f5757f7f98689fed24a975af6ec6b9395b/cluster/join.go):

```go
const numAttempts int = 3
const attemptInterval time.Duration = 5 * time.Second
```

Can we allow overriding this from the CLI?

For example:

```
rqlite -join http://0.0.0.0:4001 -join-attempts 5 -join-interval=10
```

**Comment by otoolep** [comment]
That's a reasonable request, I'll add it.

**Comment by otoolep** [comment]
https://github.com/rqlite/rqlite/pull/654

**Comment by otoolep** [comment]
Fixed  in 5.4.0.
